airbyte-config/config-models/src/main/resources/types/StandardDiscoverCatalogInput.yaml

```diff
@@ -6,9 +6,21 @@ description: information required for connection.
 type: object
 required:
   - connectionConfiguration
+  - sourceId
+  - connectorVersion
+  - configHash
 additionalProperties: false
 properties:
   connectionConfiguration:
     description: Integration specific blob. Must be a valid JSON string.
     type: object
     existingJavaType: com.fasterxml.jackson.databind.JsonNode
+  sourceId:
```

Comment:
just a comment we don't need to address in this PR: this is a smell that we need to wrap this twice.